### PR TITLE
BWS-219: + bs tests

### PR DESCRIPTION
**Pull request by Derrbal**

### CODE REVIEW

File: tests/specs/pa/bs-pa.ts [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+          });
+        let tableCell = await $('tr>td[class="styles_tableCell__C6dnh"]');
+        let tableCellValue = await tableCell.getText();
+        let fileCreated;
+        if (bsData.FileName == tableCellValue){
+           fileCreated = true;
+        }
+        else{
+           fileCreated = false;
+        }
```
Comment (what1s1ove) [inline tests/specs/pa/bs-pa.ts]:
```suggestion

           fileCreated = bsData.FileName == tableCellValue;
```

File: tests/specs/pa/bs-pa.ts [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        assert.equal(fileCreated,true);
+    }
+
+    async CheckFileDeleting() {
+        let tableCell = await $('tr>td[class="styles_tableCell__C6dnh"]');
```
Comment (what1s1ove) [inline tests/specs/pa/bs-pa.ts]:
```suggestion
        let tableCell = await $('tr>td[class="styles_tableCell__C6dnh"]');
```
bad styles. let discuss it on the call

File: tests/specs/pa/bs-pa.ts [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+        let fileCreated;
+        if (await tableCell.isExisting() == true){
+           let tableCellValue = await tableCell.getText();
+           if (bsData.FileName == tableCellValue){
+              fileCreated = true;
```
Comment (what1s1ove) [inline tests/specs/pa/bs-pa.ts]: same with if

Comment (what1s1ove) [reply tests/specs/pa/bs-pa.ts]: (fix for all)

File: tests/specs/test-files/Logo.svg [hunk 4]
```diff
@@ -0,0 +1,5 @@
+<svg width="378" height="171" viewBox="0 0 378 171" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (what1s1ove) [inline tests/specs/test-files/Logo.svg]: are you sure that you need this image
change the name to cebab case

**Review by what1s1ove — CHANGES_REQUESTED** [review]

File: tests/specs/test-files/Logo.svg (hunk 4 above)
Comment (Derrbal) [reply tests/specs/test-files/Logo.svg]: this file is required for upload testing

Comment (Derrbal) [reply tests/specs/test-files/Logo.svg]: or i can use any else .ts file

Comment (what1s1ove) [reply tests/specs/test-files/Logo.svg]: okay, use this one

Comment (Derrbal) [reply tests/specs/test-files/Logo.svg]: now the dashboard.ts file is used for testing uploads)

**Review by what1s1ove — APPROVED** [review]

**Pull request merged**